runner/runner.js

```diff
@@ -83,7 +96,7 @@ const run = async (params) => {
       })
     })
   } catch (e) {
-    config.log.error(`Error storing on IPFS network: ${e.message}`)
+    config.log.error(`Error storing on IPFS network: ${e}`)
```

Comment:
this will include the full stacktrace:
 
```suggestion
    config.log.error({ err }, `Error storing on IPFS network`)
```

Reply:
done